L1Trigger/TrackFindingTMTT/interfaceDemo/README

```diff
@@ -0,0 +1,3 @@
+The files in this subdirectory are compiled only when using the TMTT software with
```

Comment:
README files provide useful documentation, so I'm reluctant to remove them. They tend to be more up to date than twiki pages, since anyone updating the code sees them.

Reply:
I'm not against README files as a whole; I'm just against README files that present private recipes that don't apply to the official repository

Reply:
The code in interfaceDemo/ is no longer in this PR.